backend/src/test/java/reviewme/highlight/domain/HighlightedLinesTest.java

```diff
@@ -84,4 +84,23 @@ class HighlightedLinesTest {
         assertThatCode(() -> highlightedLines.addRange(invalidLineIndex, 0, 1))
                 .isInstanceOf(InvalidHighlightLineIndexException.class);
     }
+
+    @Test
+    void 하이라이트_엔티티로_변환한다() {
+        // given
+        HighlightedLines lines = new HighlightedLines("0\n11\n222");
+        lines.addRange(0, 0, 0);
+        lines.addRange(1, 0, 1);
+        lines.addRange(2, 2, 2);
+
+        // when
+        List<Highlight> highlights = lines.toHighlights(1L);
+
+        // then
+        assertThat(highlights).containsExactly(
+                new Highlight(1L, 0, new HighlightRange(0, 0)),
+                new Highlight(1L, 1, new HighlightRange(0, 1)),
+                new Highlight(1L, 2, new HighlightRange(2, 2))
+        );
```

Comment:
제안)
현재 `HighlightedLines`의 필드 lines는 하이라이트가 안쳐져있는 line까지 포함하고 있지만, 엔티티로 변환할 때는 하이라이트가 쳐져있는 부분만 걸러서 변환이 되어요.

```java
    public List<Highlight> toHighlights(long answerId) {
        return IntStream.range(0, lines.size())
                .mapToObj(lineIndex -> lines.get(lineIndex).getRanges().stream() // 이 부분
                        .map(range -> new Highlight(answerId, lineIndex, range)))
                .flatMap(Function.identity())
                .toList();
    }
```

따라서, 하이라이트가 안쳐져있는 line 걸러서 변환을 잘 했는지를 확인하는 것이 필요하다고 생각해요!

```suggestion
    @Test
    void 하이라이트_엔티티로_변환한다() {
        // given
        HighlightedLines lines = new HighlightedLines("0\n11\n222");
        lines.addRange(0, 0, 0);
        lines.addRange(2, 2, 2);

        // when
        List<Highlight> highlights = lines.toHighlights(1L);

        // then
        assertThat(highlights).containsExactly(
                new Highlight(1L, 0, new HighlightRange(0, 0)),
                new Highlight(1L, 2, new HighlightRange(2, 2))
        );
```